CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.37.0] - 2021-11-30
```

Comment:
shouldn't this be a patch?

Reply:
No, adding features should bump minor version.